gcc/tests/bats/test.bats

```diff
@@ -0,0 +1,50 @@
+setup() {
+  TEST_PKG_PATH="/hab/pkgs/$TEST_PKG_IDENT"
+  BINUTILS_DEP=$(grep binutils /hab/pkgs/$TEST_PKG_IDENT/DEPS)
+  GLIBC_DEP=$(grep glibc /hab/pkgs/$TEST_PKG_IDENT/DEPS)
+}
+
+# gcc,g++ and cpp are wrapped with shell scripts to allow us to carefully set the
+# environment and flags passed to the underlying command. Testing core/gcc is best
+# tested by building additional software with the resulting package. The following
+# asserts that we didn't have an unexpected failure or accidental change in our
+# build process by asserting:
+#
+# That the commmand are shell scripts
+# That the `.real` command exists and is executable
+# That the final line in the script is an exec to become the real command
+
+@test "gcc is wrapped" {
+  [ "$(file -bi "$TEST_PKG_PATH"/bin/gcc)" == "text/x-shellscript; charset=us-ascii" ]
```

Comment:
The output of `file` seems to vary a lot on different distributions. I think if these tests always run in a studio we can be comfortable it will be consistent.

Reply:
Yup, in this case we should be good. For a little more context on our adventures with `file`,  read habitat-sh/habitat#6189